support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardFormField.tsx

```diff
@@ -20,11 +20,11 @@ const getClass = (props: PropTypes) => {
 	return '';
 };
 
-export function StripeCardFormField(props: PropTypes) {
+export function StripeCardFormField(props: PropTypes): JSX.Element {
 	return (
 		<>
 			<div className="ds-stripe-card-form-field">{props.label}</div>
-			{props.hint || null}
+			{props.hint ?? null}
```

Comment:
I think we might be alright without the `??` here and just render `{props.hint}`

Reply:
Ah good call - done!